urgent-care/src/test/java/gov/va/api/health/urgentcare/service/api/swaggerexamples/SwaggerMetadata.java

```diff
@@ -0,0 +1,23 @@
+package gov.va.api.health.urgentcare.service.api.swaggerexamples;
+
+import gov.va.api.health.autoconfig.configuration.JacksonConfig;
+import gov.va.api.health.r4.api.resources.Capability;
+import java.io.IOException;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class SwaggerMetadata {
+
+  static final Capability SWAGGER_EXAMPLE_METADATA;
+
+  static {
+    try {
+      SWAGGER_EXAMPLE_METADATA =
+          JacksonConfig.createMapper()
```

Comment:
You can use `@SneakyThrows` if you put this inside a static method

Reply:
Updated to be sneaky.